mitodl/salt-ops
# config: Add GTM and GA keys for MITxOnline Prod

#### What are the relevant tickets?
#1462 

#### What's this PR do?
- Adds values for GTM related keys:
    - `GOOGLE_TRACKING_ID `
    - `GOOGLE_TAG_MANAGER_ID `


#### How should this be manually tested?
- Once these config values are in place for the Prod environment of MITxOnline, we should start seeing the page events (In [Google Analytics](https://analytics.google.com/))
- On the other hand, At least we should see a network request to google analytics e.g. something like `https://www.google-analytics.com/j/collect?` whenever we visit a page.
